app/presenters/support_interface/provider_relationships_diagram.rb

```diff
@@ -78,19 +84,29 @@ def providers
     end
 
     def translate_training_permissions(permission)
-      "#{value_indicator(permission.training_provider_can_view_safeguarding_information)} view safeguarding "\
-      "#{value_indicator(permission.training_provider_can_view_diversity_information)} view diversity "\
-      "#{value_indicator(permission.training_provider_can_make_decisions)} make decisions"
+      return 'Permissions not setup' if permissions_not_setup(:training_provider, permission)
+
+      "can #{build_permissions_string(:training_provider, permission)} for courses ratified by"
     end
 
     def translate_ratifying_permissions(permission)
-      "#{value_indicator(permission.ratifying_provider_can_view_safeguarding_information)} view safeguarding "\
-      "#{value_indicator(permission.ratifying_provider_can_view_diversity_information)} view diversity "\
-      "#{value_indicator(permission.ratifying_provider_can_make_decisions)} make decisions"
+      return 'Permissions not setup' if permissions_not_setup(:ratifying_provider, permission)
+
+      "can #{build_permissions_string(:ratifying_provider, permission)} for courses run by"
     end
 
     def value_indicator(permission_value)
       permission_value ? '✅' : '❌'
     end
+
+    def permissions_not_setup(provider_type, permission)
```

Comment:
I think `permissions_not_setup` is not checking the right things. It's perfectly possible for either side of the relationship to have no permission to do anything (other than see the application), while the relationship has been properly set up. I think 'set up', in this instance, means that the training provider has been through the 'organisational permissions' flow.

There are other ways of detecting that this has not happened, such as checking if the relationship is `valid?` (which means at least one side has permission for each rule) but the simplest way is  checking for the presence of `ProviderRelationshipPermissions#setup_at`.